File: modulo_1_flota/internal/domain/route.go
package domain

import (
	"time"
)

type Route struct {
	ID          string    `json:"id"`
	Name        string    `json:"name"`
	Description string    `json:"description"`
	Stops       []Stop    `json:"stops,omitempty"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
}

File: modulo_4_admin/src/core/entities/Route.ts
export interface Route {
  id: string;
  name: string;
  description: string;
  stops: Stop[];
  createdAt: Date;
  updatedAt: Date;
}

export interface Stop {
  id: string;
  name: string;
  latitude: number;
  longitude: number;
  order: number;
  routeId: string;
  createdAt: Date;
  updatedAt: Date;
}
